.env.development

```diff
@@ -19,3 +19,5 @@ REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 SEGMENT_KEY=''
 SITE_NAME=localhost
 USER_INFO_COOKIE_NAME='edx-user-info'
+APP_ID=
+MFE_CONFIG_API_URL=
```

Comment:
This is what edX contributors added to the dev envs in related runtime config PRs. The frontend app still calls `http://apps.local.overhang.io:1998/api/mfe_config/v1?mfe=mogc-partners` which is proxied to `http://local.overhang.io:8000`.